### PR TITLE
Feat/algosigner wallet switch account

### PR DESCRIPTION
**Pull request by vuvoth**
Algosigner Wallet switch account. 

image

### CODE REVIEW

File: wallet/src/App.vue [hunk 1]
```diff
@@ -109,6 +125,10 @@ export default defineComponent({
     };
   },
   methods: {
+    async setCurrentAddress(e: any) {
```
Comment (rmeena840) [inline wallet/src/App.vue:128]: What if undefined value came here? We should have safe check here.
`If(e.target.value){// do something}`

**Review by rmeena840 — APPROVED** [review]

**Review by rmeena840 — APPROVED** [review]
Pre-approving. Just look at one comment I posted.

File: wallet/src/App.vue [hunk 2]
```diff
@@ -109,6 +125,10 @@ export default defineComponent({
...
+      this.currentAddress = e.target.value;
```
Comment (vuvoth) [inline wallet/src/App.vue]:
```suggestion
      if (e.target.value) {
          this.currentAddress = e.target.value;
      }
```

File: wallet/src/App.vue (hunk 1 above)
Comment (vuvoth) [reply wallet/src/App.vue:128]: Done

**Pull request merged**